# web chain message

int32 plan_step
string request_type
string[] parameters # [prepro, proce, default]
#default_type
float32 duration


